### PR TITLE
Update requirements

### PR DESCRIPTION
**Pull request by joshmoore**
This fixes travis for subsequent testing by (the likes of) https://github.com/openmicroscopy/ome-ansible-molecule-dependencies/pull/3

### CODE REVIEW

File: tests/requirements.yml [hunk 1]
```diff
@@ -1,6 +1,3 @@
-- name: openmicroscopy.omero-common
-  src:  https://github.com/openmicroscopy/ansible-role-omero-common.git
-- name: openmicroscopy.omego
-  src:  https://github.com/openmicroscopy/ansible-role-omego.git
-- name: openmicroscopy.omero-web
-  src:  https://github.com/manics/ansible-role-omero-web.git
+- role: openmicroscopy.omero-common
```
Comment (manics) [inline tests/requirements.yml]: Minor code-style issue: I've been using `- src: openmicroscopy.role` instead of `role: ...`

File: playbook.yml [hunk 2]
```diff
@@ -4,7 +4,7 @@
   roles:
 
     - role: openmicroscopy.omero-web
-      omero_web_release: 5.3.0-m9
+      omero_web_release: 5.3.2
```
Comment (manics) [inline playbook.yml]: Optional: in theory `omero_web_release: 5.3` should work

**Comment by manics** [comment]
Both comments are optional

**Comment by joshmoore** [comment]
Done. Propose to tag as 0.1.1 once this is green again.

**Pull request merged**